modules/localgov_campaigns_paragraphs/localgov_campaigns_paragraphs.module

```diff
@@ -46,6 +46,14 @@ function localgov_campaigns_paragraphs_theme($existing, $type, $theme, $path) {
       'template' => 'paragraph--localgov-fact-box',
       'base hook' => 'paragraph',
     ],
+    'paragraph__localgov_featured_teaser' => [
+      'template' => 'paragraph--localgov-featured-teaser',
```

Comment:
For reference, when the template name matches the theme hook name, we can skip the *template* key.  In this case paragraph__localgov_featured_teaser will result in a template name of paragraph--localgov-featured-teaser.html.twig, so the template declaration is redundant.  We have some [examples of these](https://github.com/localgovdrupal/localgov_paragraphs/blob/3e03b3070237d1ecb6995a7e35bcaa639e54b0a9/modules/localgov_homepage_paragraphs/localgov_homepage_paragraphs.module#L23) in the localgov_homepage_paragraphs module.

Reply:
Thanks Adnan! You're right, 'template' key is not needed for these 2 paragraphs. Removed.